packages/terra-date-picker/tests/jest/__snapshots__/DateInput.test.jsx.snap

```diff
@@ -153,7 +153,7 @@ exports[`correctly applies the theme context className 1`] = `
                   >
                     <svg
                       aria-hidden="true"
-                      className="tui-Icon icon icon-svg"
+                      className="tui-Icon icon orion-fusion-theme icon-svg"
```

Comment:
These changes (including the terra-menu screenshot update) are already being added in https://github.com/cerner/terra-framework/pull/1153.